dashboard/src/main/resources/css/dashboard.css

```diff
@@ -10,6 +10,11 @@
    background-color: gray;
    font-weight: bold;
  }
+
+ p.DEPENDENCY_TREE_NODE {
```

Comment:
p, not li?

Reply:
We need an HTML tag to hold the label (title) to show the mouse-hover. Adding title to li would make the scope of the li too big.